CONTRIBUTING.md

```diff
@@ -242,6 +244,26 @@ $ cabal run flora-cli -- import-index ~/.cabal/packages/cardano/01-index.tar.gz
   --repository "cardano"
 ```
 
+### Live Eventlogs
```

Comment:
Great stuff. Did you try to run flora-server in its docker compose and reach the live-eventlog docker compose?

Reply:
Ah sorry I saw that we can't do that on macOS. No worries!